Dockerfile

```diff
@@ -1 +1,37 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java11:latest
+
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
+      IMAGE_SOURCE=https://github.com/ArkCase/ark_base_java11_tomcat9 \
+      MAINTAINER="Armedia LLC"
+
+RUN useradd  --system --user-group tomcat
+
+ENV TOMCAT_VERSION=9.0.22  \
+    TOMCAT_MAJOR_VERSION=9
```

Comment:
Can the ENV lines below be combined with these?

Reply:
Yeah updated with single ENV command.